gammapy/modeling/models/core.py

```diff
@@ -791,6 +791,57 @@ def reassign(self, dataset_name, new_dataset_name):
         models = [m.reassign(dataset_name, new_dataset_name) for m in self]
         return self.__class__(models)
 
+    def plot_spatial(self, ax=None, wcs=None, plot_extent=True, **kwargs):
+        """"Plot spatial models on a given geom
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`, optional
+            Axis
+        wcs: `~astropy.wcs.WCS`
```

Comment:
I think passing an optional wcs is not needed here, because users can always create a `WcsAxes` outside. The question is what to do for the case of `ax=None`. For `RegionGeom.plot_region()` we decided to just create a default wcs. This you can re-use here:
```python3
from gammapy.maps import RegionGeom
from gammapy.utils.regions import make_region
import numpy as np

regions = []

lons = np.random.uniform(-5, 5, 100)
lats = np.random.uniform(-3, 3, 100)
radii = np.random.uniform(0.1, 0.7, 100)

for lon, lat, radius in zip(lons, lats, radii):
    region = make_region(f"icrs;circle({lon}, {lat}, {radius})")
    regions.append(region)

geom = RegionGeom.from_regions(regions)
geom.plot_region()
```
To avoid code duplication you could possibly re-use the `RegionGeom.plot_region()` as well...

Reply:
`RegionGeom.plot()` creates by default "minimal equivalent WCS geometry", which is good for one model. But for plotting multiple models, a default all-sky option is better no?